app/assets/stylesheets/home.css.scss

```diff
@@ -0,0 +1,30 @@
+.text-with-side-image {
+  @include outer-container;
+  @include span-columns(12);
+  .side-image {
+    @include span-columns(3 of 12);
+    border: outset;
+    border-width: thick;
+    margin-right: 117px;
+    margin-top: 100px;
+    .block-text {
+      @include span-columns (5 of 12);
+      text-align: justify;
+    }
+  }
+}
+
+.block-text {
+  @include span-columns(5);
+  text-align: justify;
+}
+
+.notice {
+  @include span-columns(4);
+  background-color: $light-tan;
+  text-align: justify;
```

Comment:
Properties should be ordered background-color, margin-top, outline-color, padding-left, padding-right, text-align